main/Repl.hs

```diff
@@ -180,7 +181,13 @@ completer = Prefix (wordCompleter comp) defaultMatcher
 
 shell :: (MonadNix e m, MonadIO m, MonadException m) => Repl e m a -> m ()
 shell pre = flip evalStateT initState $
-    evalRepl "hnix> " cmd options completer pre
+#if MIN_VERSION_repline(0, 2, 0)
+    evalRepl (return prefix() cmd options Nothing completer pre
```

Comment:
This line causes the build to fail with a syntax error.

Reply:
Yes, sorry. I've fixed it in https://github.com/haskell-nix/hnix/pull/409